.license_plist.yml

```diff
@@ -0,0 +1,24 @@
+# Documentation https://github.com/mono0926/LicensePlist
+
+options:
+  outputPath: ./wire-ios/Wire-iOS/Resources/Settings.bundle
+  cartfilePath: Cartfile
+  force: true
+  addVersionNumbers: true
+  suppressOpeningDirectory: true
+  failIfMissingLicense: true
+  sandboxMode: true
+
+manual:
```

Comment:
In the current solution, we also had some manual licenses for open source repos we used that aren't actually pulled into the project, for instance the emoji database that is used for reactions. I guess we can add those licenses here right?